main.go

```diff
@@ -283,6 +282,8 @@ func main() {
 	archiveZipPath := filepath.Join(cfg.OutputDir, cfg.ArtifactName+".xcarchive.zip")
 	ideDistributionLogsZipPath := filepath.Join(cfg.OutputDir, "xcodebuild.xcdistributionlogs.zip")
 
+	xcarchiveZipPath := filepath.Join(os.Getenv("BITRISE_DEPLOY_DIR"), cfg.ArtifactName+".xcarchive.zip")
```

Comment:
Can you ensure that the zip name is unique by generating a custom artifact name? Also it seems cfg.ArtifactName is not validated to be not empty.

Reply:
Discussed the unique name and agreed that is not the best user experience. Still think that should handle the case if the artifact name is empty.